internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -111,10 +105,9 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) openAIToolsToBedrockToolC
 	bedrockReq.ToolConfig = &awsbedrock.ToolConfiguration{}
 	tools := make([]*awsbedrock.Tool, 0, len(openAIReq.Tools))
 	for _, toolDefinition := range openAIReq.Tools {
-		toolType := (string)(toolDefinition.Type)
 		tool := &awsbedrock.Tool{
 			ToolSpec: &awsbedrock.ToolSpecification{
-				Name:        &toolType,
+				Name:        &toolDefinition.Function.Name,
```

Comment:
i believe this is dangerous as this points to a loop var. you should extract the name in the local variable inside loop

Reply:
basically this feels unnecessary change